aws/logs_monitoring/release.sh

```diff
@@ -41,8 +41,18 @@ if [ "$ACCOUNT" = "prod" ]; then
     BUCKET="datadog-cloudformation-template"
 fi
 
+function awe-login() {
```

Comment:
What does `awe` stand for here? Is this supposed to be `aws-login`?

Reply:
Ahh good point. I have this command in my local bash and I reused the name. I read it as 'aws environment'. I wanted to avoid confusion with it being an official tool like 'aws' or 'aws-vault'. I can change it though if you feel it's neater.